src/cli/cmd/add/mod.rs

```diff
@@ -118,6 +118,7 @@ async fn infer_flake_input_name_url(
     flake_ref: String,
     input_name: Option<String>,
 ) -> color_eyre::Result<(String, url::Url)> {
+    let flake_ref = flake_ref.trim_end_matches('/');
```

Comment:
I believe `1.0.0-rc34/asd/////` is indeed valid semver, so this behavior may be incorrect.

Reply:
Doesn't look like it:

```
vin@scadrial ~/.c/c/tmp-9pGulc (main) 0.1s> cat src/main.rs
#![allow(warnings)]
fn main() {
    let v = "1.0.0-rc34/asd/////";
    dbg!(semver::Version::parse(v));
}

vin@scadrial ~/.c/c/tmp-9pGulc (main) 0.1s> cargo r --quiet
[src/main.rs:4] semver::Version::parse(v) = Err(
    Error("unexpected character '/' after pre-release identifier"),
)
```

Reply:
Whoa! Nice. I guess what I checked was less strict. 😓